### PR TITLE
Fix package inclusions

### PR DESCRIPTION
**Pull request by jakirkham**
Fixes https://github.com/keflavich/image_registration/issues/15

Package the license file and `astropy_helpers`.

### CODE REVIEW

File: MANIFEST.in [hunk 1]
```diff
@@ -1,2 +1,23 @@
...
+include astropy_helpers/ez_setup.py
+include astropy_helpers/ah_bootstrap.py
+
+recursive-include astropy_helpers/astropy_helpers *.py *.pyx *.c *.h *.rst
+recursive-include astropy_helpers/astropy_helpers.egg-info *
+# include the sphinx stuff with "*" because there are css/html/rst/etc.
+recursive-include astropy_helpers/astropy_helpers/sphinx *
+
+prune astropy_helpers/build
+prune astropy_helpers/astropy_helpers/tests
```
Comment (jakirkham) [inline MANIFEST.in]: Copied this text verbatim from astropy. Hope that is ok. Happy to add attribution or whatever is necessary.

**Comment by keflavich** [comment]
Thanks!  Will merge after tests complete.  I'll need to make a new release too...

**Comment by jakirkham** [comment]
I think also `ah_bootstrap.py` needs to be added to the top-level repo or at least that is how it was done in astropy. Not sure how we would like to address that here.

**Comment by jakirkham** [comment]
cc @bsipocz

**Comment by jakirkham** [comment]
Ok just did a quick test by generating the `sdist`, decompressing it, and running `pip install .` within the decompressed version. That seemed to work fine.

**Comment by jakirkham** [comment]
Seems like `matplotlib` is required to import `image_registration`. Should it be listed as a required dependency?

**Comment by keflavich** [comment]
@jakirkham yes, it should be required, could you add it here?

**Comment by jakirkham** [comment]
Sure. Also should `FITS_tools` be added too? It shows up in `REQUIREMENTS`, but I don't know if it is a hard requirement or not.

**Comment by keflavich** [comment]
I *think* it's a soft requirement, but that knowledge (i.e., in my head...) is possibly out of date

File: MANIFEST.in (hunk 1 above)
Comment (bsipocz) [reply MANIFEST.in]: Looks good, though a few more things may be still needed: https://github.com/astropy/package-template/blob/rendered/MANIFEST.in#L10

**Comment by jakirkham** [comment]
So I added some code that just reads in `REQUIREMENTS` and adds it to `install_requires`. Please let me know if it is to your liking.

Tried importing `image_registration` without `FITS_tools` and that worked fine. Looks like `FITS_tools` is actually vendored. Should it be dropped from `REQUIREMENTS` then?

**Comment by bsipocz** [comment]
I'm not sure how much up to that this with ``package-template``, probably a few other things could be copied over.

**Comment by jakirkham** [comment]
There seem to be a bunch of builds that fail due to AstroPy's Python version constraints (i.e. needs 3.5 or newer for Python 3). Not sure why an older AstroPy is not used in those cases.

One build fails as NumPy 1.9.3 doesn't have `cbrt` (added in 1.10.0).

While these are issues, they seem more to be CI configuration issues that are not related to this PR's content. The only remaining question I have is whether to drop `FITS_tools` as a requirement since it is vendored and only the vendored copy is used.

**Comment by keflavich** [comment]
when you say 'vendored', you mean packaged within `image_registration`?  

I'm going to work on another PR to fix up the travis grid, etc.  I'll probably merge this (locally?) along the way.

**Comment by jakirkham** [comment]
> when you say 'vendored', you mean packaged within `image_registration`?

Yes, this is [it]( https://github.com/keflavich/image_registration/tree/30ac03b9294ce9aeda82785c500c4f8d81d5fdcd/image_registration/FITS_tools ), right? Though there is [this interesting bit]( https://github.com/keflavich/image_registration/blob/30ac03b9294ce9aeda82785c500c4f8d81d5fdcd/image_registration/FITS_tools/hcongrid.py#L2 ). So maybe `FITS_tools` is still required?

> I'm going to work on another PR to fix up the travis grid, etc. I'll probably merge this (locally?) along the way.

SGTM. Thanks for your help.

**Comment by keflavich** [comment]
Yes, FITS_tools is required.  Originally, I had copied code around instead of creating independent packages.  I'm working on updating the FITS_tools package too.

File: MANIFEST.in (hunk 1 above)
Comment (keflavich) [reply MANIFEST.in]: Yes, I think so?  Sorry, I couldn't find this comment earlier.

Comment (jakirkham) [reply MANIFEST.in]: Ok, since this is already merged. Would just copy this `recursive-include docs *` and add it here.

Comment (keflavich) [reply MANIFEST.in]: Thanks, I'll add that as I'm making the pypi release

**Pull request merged**